meta/documents/user_guide/de/updating.md

```diff
@@ -0,0 +1,62 @@
+
+## Aktualisierung auf die neueste Version des Wayfair Plugins
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Plugins` >>` Plugin Set Übersicht`
+    * **Wählen Sie nicht die veraltete Option `Plugin-Übersicht.`**
+
+    * Auf dem Bildschirm sollte eine Liste der Plugin-Sets mit jeweils einer Reihe von Schaltflächen angezeigt werden.
+
+        ![plugins_menu_plugin_set_overview](../../../images/de/plugins_menu_plugin_set_overview.png)
+
+2. Suchen Sie das Plugin-Set, in dem das Wayfair-Plugin installiert ist.
+
+    * Die Liste der Plugins für das Plugin-Set ist hinter der Schaltfläche `Plugin-Set bearbeiten` versteckt.
+        ![linked clients](../../../images/de/plugin_sets_linked_clients.png)
+
+3. Klicken Sie auf die Schaltfläche `Plugin-Set bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das Plugin-Set.
+
+4. Stellen Sie sicher, dass auf der Seite eine Zeile mit dem Namen "Wayfair" vorhanden ist und dass auch die Spalte `Bereitgestellt` ausgefüllt ist, um anzuzeigen, dass das Plugin verwendet wird.
+    ![wayfair plugin needs update](../../../images/de/installation/wayfair_plugin_needs_update.png)
+
+5. Bewegen Sie den Mauszeiger über die Schaltfläche `Aktualisieren` ![update button](../../../images/common/button_update.png) in der Spalte `Aktionen`.
+
+5. Wenn auf der Schaltfläche die Meldung `Keine Aktualisierung verfügbar` angezeigt wird, verfügen Sie bereits über die neueste Version des Wayfair-Plugins. **Fahren Sie nicht mit dieser Anleitung fort.** Andernfalls sollte `Aktualisierung verfügbar` und eine Versionsnummer angezeigt werden.
+
+      ![no update](../../../images/de/installation/no_update.png)
+
+6. Klicken Sie auf die Schaltfläche `Aktualisieren`. Es wird eine Seite mit dem Wayfair-Logo, Informationen zum Wayfair-Plugin und einer Schaltfläche `Aktualisieren` oben angezeigt.
+
+    ![update page](../../../images/de/installation/update_page.png)
+
+7. Überprüfen Sie die Informationen auf der Seite.
+
+9. Wählen Sie im Dropdown-Menü `Version auswählen` die Option [die neueste Plugin-Version](https://github.com/wayfair-contribs/plentymarkets-plugin/releases), falls diese noch nicht ausgewählt ist.
+
+    ![update selector](../../../images/de/installation/update_version_selection.png)
+
+10. Klicken Sie oben auf der Seite auf die Schaltfläche `Aktualisieren` ![update button](../../../images/de/installation/update_button.png). Nach einer kurzen Pause werden Sie zu den Details des Plugin-Sets weitergeleitet.
+
+11. Stellen Sie sicher, dass auf der Seite eine Zeile mit dem Namen `Wayfair` und der installierten Version als gewünschte Version vorhanden ist
+
+12. Klicken Sie in der Spalte `Aktiv` für die Zeile `Wayfair` auf den Schalter, damit er sich in die aktivierte Position auf der rechten Seite bewegt. **Wenn das Wayfair-Plugin nicht aktiviert wird, sind dessen Funktionen nicht verfügbar**.
+
+13. Stellen Sie sicher, dass die Zeile des Wayfair-Plugins so aussieht.

```

Comment:
Verify this translation with Falco. Looks inaccurate
This should say "Verify that the Wayfair plugin's row looks like this:"

Reply:
The translations that were add have be verified by Falco